mantidproject/mantid
# Visualise off detector peaks from Predict Peaks

**Note: This must not be merged before #18921**

This adds the ability to view predicted peaks which do not land on any detector to the instrument view. The main changes are:

 - Update the IDF for WISH to include an extended detector space definition.
 - Add the ability to zoom out on an unwrapped view in the instrument view.
 - Add the ability to reset the zoom in an unwrapped view.

**To test:**

For testing with WISH you'll need a machine with lots of memory. Alternatively you can add a extended detector space component to an instrument of your choice.

 - Load some single crystal raw data for WISH and open the instrument view
 - Add some single crystal peaks
 - Use `FindUBUsingFFT` to get a UB matrix
 - Use `PredictPeaks` to predict the position of peaks and check the `PredictPeaksOffDetectors` option.
 - Drag the generated workspace onto the instrument view
 - Zoom in/out with left/right click & drag to view peaks.
 - Ctrl+Right Click resets the zoom.

Fixes #18923.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
